core/controlplane/config/templates/cluster.yaml

```diff
@@ -1124,6 +1124,13 @@ worker:
 #kubeResourcesAutosave:
 #  enabled: false
 
+# When enabled, all nodes will forward journald logs to AWS CloudWatch.
+# It is disabled by default.
+#cloudWatchLogging:
+# enabled: false
+# imageWithTag: jollinshead/journald-cloudwatch-logs:0.1
```

Comment:
AFAIK this is not how images are generally handled in kube-aws, see example for dex image